appeals/web/src/client/components/file-uploader/_server-actions.js

```diff
@@ -137,9 +135,29 @@ const serverActions = (uploadForm) => {
 		}
 	};
 
+	/**
+	 * Deletes session upload info for a single uncommitted file (does not remove file from blob storage, only deletes metadata from session)
+	 * @param {string} guid
+	 */
+	const deleteUncommittedFileFromSession = async (guid) => {
```

Comment:
Hey @james-hill-kc  we may need to do the clean up elsewhere, as there may be no DELETE permissions on the servers. Probably worth wrapping in a try/catch and logging errors on the deployment.

Reply:
Looks like they should have delete permission https://github.com.mcas.ms/Planning-Inspectorate/infrastructure-tooling/pull/76/files